linode/manager
# M3-5772: Cleanup Button story

## Description

Tasks:

- [x] Remove examples of "Cancel" and "Remove" buttons (also remove related code from the actual component)
- [x] Remove "Loading with Text" story
- [x] Remove references to "compact" and replace with the code for "superCompact"
- [x] Add example of a compact button for secondary buttons only
- [x] Document use cases for the different variants
- [x] Add details to `ArgsTable`

Future tasks to be addressed in a separate PR:

- Figure out why the Component API Button's generated code isn't accurate

## How to test

- Check the buttons throughout the app
- Run `yarn storybook` to see the documentation for Buttons (it's nested inside the UI Elements folder)
